.github/workflows/main.yml

```diff
@@ -411,15 +412,16 @@ jobs:
     - name: Install conda dependencies
       # See https://github.com/conda-forge/boost-cpp-feedstock/issues/41 for why we
       # use boost-cpp rather than boost from conda-forge
-      run: mamba install -q scons numpy cython ruamel.yaml boost-cpp eigen yaml-cpp
-        h5py pandas pytest mkl mkl-devel
+      # Install SCons >=4.4.0 to make sure that MSVC_TOOLSET_VERSION variable is present
+      run: |
+        mamba install -q '"scons>=4.4.0"' numpy cython ruamel.yaml boost-cpp eigen yaml-cpp h5py pandas pytest
       shell: pwsh
     - name: Install typing_extensions for Python 3.7
       if: matrix.python-version == '3.7'
       run: mamba install -q typing_extensions
     - name: Build Cantera
       run: scons build system_eigen=y system_yamlcpp=y logging=debug
-        msvc_version=${{ matrix.vs-toolset }} f90_interface=n debug=n --debug=time -j2
+        msvc_toolset_version=${{ matrix.vs-toolset }} msvc_version=14.3 f90_interface=n debug=n --debug=time -j2
```

Comment:
Does it make sense to leave off the `msvc_version` here, so that it doesn't have to be changed when GitHub upgrades their images?